storybook/src/components/Field/Field.stories.tsx

```diff
@@ -3,7 +3,7 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { ErrorMessage, Label, TextInput } from '@amsterdam/design-system-react'
+import { ErrorMessage, Label, TextInput } from '@amsterdam/design-system-react/src'
```

Comment:
This isn't really necessary, only the controls for Field are generated, right? We can move the `Paragraph` import to the regular import as well.

Reply:
Ah yes, didn’t realise that distinction. Updated all components in one fell swoop.